nodejs-example/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejs-example",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "To generate Canvas authentication tokens from NodeJS you can use the following snippet:",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
kill?

Reply:
and maybe add a `run` script?

Reply:
assume you mean `start` script to run `node index.js`, done